src/test/skript/tests/syntaxes/expressions/ExprBeaconValues.sk

```diff
@@ -0,0 +1,22 @@
+on load:
+	set {_loc} to spawn of world "world" ~ vector(5,5,5)
+	set block at {_loc} to beacon
+
+
+on beacon activate:
+	set primary beacon effect to haste
+	assert primary beacon effect is haste with "Set beacon primary effect to haste"
+	clear primary beacon effect
+	assert primary beacon effect is not set with "Clear beacon primary effect"
+
+	set secondary beacon effect to regeneration
+	assert secondary beacon effect is regeneration with "Set beacon secondary effect to regeneration"
+	clear secondary beacon effect
+	assert secondary beacon effect is not set with "Clear beacon secondary effect"
+
+	set beacon range to 10
+	assert beacon range is 10 with "Set beacon range to 10"
+	add 5 to beacon range
+	assert beacon range is 15 with "Add 5 to beacon range"
+
+	set event-block to air
```

Comment:
i'm a little concerned about this test
Can you guarantee the physical beacon is the one triggering this event, that's it's only being triggered for this test, and that it's setting the right block to air?
I'm not even sure it's triggering at all except maybe due to the junit test?